src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -153,6 +153,23 @@ private void recordNewSentEmail(String subject,
 
         moveJobToGroupModeTaskQueue();
         moveJobToAddressModeTaskQueue();
+
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email will be sent within an hour to "
+                            + addressReceiverListString + "," + groupReceiverListFileKey;
+            statusToUser.add(new StatusMessage("Email will be sent within an hour to "
+                    + addressReceiverListString + ", " + groupReceiverListFileKey, StatusMessageColor.SUCCESS));
+
+        } else if (addressModeOn) {
+            statusToAdmin = "Email wll be sent within an hour to " + addressReceiverListString;
+            statusToUser.add(new StatusMessage("Email will be sent within an hour to " + addressReceiverListString,
+                         StatusMessageColor.SUCCESS));
+
+        } else {
+            statusToAdmin = "Email will be sent within an hour to " + groupReceiverListFileKey;
+            statusToUser.add(new StatusMessage("Email will be sent within an hour to "
+                        + groupReceiverListFileKey, StatusMessageColor.SUCCESS));
```

Comment:
@Mynk96 Also, in `moveJobToGroupModeTaskQueue` and `moveJobToAddressModeTaskQueue`, you get the `if-else` for free